### ISSUE TITLE
[Bug?] Download location file structure + nomedia

### ISSUE THREAD

**Issue by sudoshindo** [rebelonion/Dantotsu#398]
As of the current version (**3.1.0**) when I select a Download Location it seems to create the `.nomedia` file in the wrong place.

For example, if I select:

`/sdcard/Videos`

(SD card path is simplified for the purposes of this example, not the actual path)

It creates the `.nomedia` here:

`/sdcard/Videos/.nomedia`

And creates the Dantotsu directory here:

`/sdcard/Videos/Dantotsu`

I believe this is wrong because `.nomedia` should be placed inside the Dantotsu folder.

If `.nomedia` is placed one level above the Dantotsu folder then it creates the side effect of hiding any media files present in the parent folder.

**Comment by rebelonion** [comment]
Thx for letting me know. In testing I kinda assumed people would make a new folder, but yeah I'll move it inside the dantotsu folder. 
For now you should be able to move it manually

**Comment by sudoshindo** [comment]
Yeah, if I use an app that downloads video files, I like telling all of them to use the same folder (NewPipe, Seal, Dantotsu).

One other thing to consider, people might want to create a Download Location that looks like this (I also considered maybe using this structure):

`/sdcard/Dantotsu`

`/sdcard/Dantotsu/Anime`

But because of how the app currently works this is not possible, instead it would create:

`/sdcard/Dantotsu/Dantotsu`

`/sdcard/Dantotsu/Dantotsu/Anime`

Just throwing this in as another case to consider.
